vscode-web/main.tf

```
@@ -97,6 +97,30 @@ variable "settings" {
   default     = {}
 }
 
+variable "offline" {
+  type        = bool
+  description = "Just run VS Code Web in the background, don't fetch it from GitHub"
```

I forgot we fetch this from microsoft.com, not github.com.  I see also that we use the terminology "the internet" in the `use_cached` setting so I think we can use that here as well.

```suggestion
  description = "Just run VS Code Web in the background, don't fetch it from the internet."
```